redisson-spring-boot-starter/src/main/java/org/redisson/spring/starter/RedissonAutoConfiguration.java

```diff
@@ -55,15 +56,21 @@
 @EnableConfigurationProperties({RedissonProperties.class, RedisProperties.class})
 public class RedissonAutoConfiguration {
 
-    @Autowired
     private RedissonProperties redissonProperties;
-    
-    @Autowired
     private RedisProperties redisProperties;
-    
-    @Autowired
+    private List<RedissonAutoConfigurationCustomizer> redissonAutoConfigurationCustomizers;
     private ApplicationContext ctx;
-    
+
+    public RedissonAutoConfiguration(RedissonProperties redissonProperties,
+                                     RedisProperties redisProperties,
+                                     ObjectProvider<RedissonAutoConfigurationCustomizer> redissonAutoConfigurationCustomizers,
```

Comment:
Don't use ObjectProvider since it breaks Spring 4.2 compatibility required to Spring Boot 1.3, which is supported. I think autowire of RedissonAutoConfigurationCustomizer should be enough.